image_recognition_tensorflow/test/models/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
would `models/*` in a `.gitignore` on parent level not be better?

Reply:
I like that there is a models folder on checkout